src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -300,13 +289,13 @@ const styles = StyleSheet.create({
     position: 'absolute',
     opacity: opacityHeaderAnimation
   }),
-  image: {
+  image: (followType) => ({
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
-    backgroundColor: 'lightgrey',
+    backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
     marginRight: 8
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `image` style now accepts a `followType` parameter and sets the background color based on it. This change aligns with the PR's purpose of visually distinguishing between signed and anonymous follows.
```diff
- image: {
-    width: normalize(48),
-    height: normalize(48),
-    borderRadius: normalize(24),
-   backgroundColor: 'lightgrey',
-    marginRight: 8
- },
+ image: (followType) => ({
+    width: normalize(48),
+    height: normalize(48),
+    borderRadius: normalize(24),
+   backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
+    marginRight: 8
+ }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->